complete/src/main/resources/static/app.js

```diff
@@ -18,100 +18,134 @@ function refreshTimeTable() {
         var theadByRoom = $("<thead>").appendTo(timeTableByRoom);
         var headerRowByRoom = $("<tr>").appendTo(theadByRoom);
         headerRowByRoom.append($("<th>Timeslot</th>"));
-        $.each(timeTable.roomList, function (index, room) {
-            headerRowByRoom.append($("<th>"
-                    + "<span>" + room.name + "</span>"
-                    + "<button id=\"deleteRoomButton-" + room.id + "\" type=\"button\" class=\"ml-2 mb-1 btn btn-light btn-sm p-1\">"
-                    + "<small class=\"fas fa-trash\"></small>"
-                    + "</button>"
-                    + "</th>"));
-            $("#deleteRoomButton-" + room.id).click(function() {
+        $.each(timeTable.roomList, (index, room) => {
+            const id = uuid();
```

Comment:
Why introduce the uuid for the id? Room id is guaranteed to be a Long. so deleteRoomButton-" + room.id is safe, no?

Reply:
I was being extra paranoid, in case an attacker found a flaw either in the database system or somewhere else within the backend, allowing them to replace id's with malicious content. However, by then we have bigger problems to worry about. I was able to remove these id's though.